### PR TITLE
 Added spelling game prototype 

### PR DESCRIPTION
**Pull request by ortegacmanuel**
spelspel

korekto

### CODE REVIEW

File: app/models/spelling_group.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "#{name}"
+  end
+
+
+  def inline_forms_attribute_list
+    @inline_forms_attribute_list ||= [
+      [ :name , "name", :text_field ],
+      [ :caption , "caption", :text_field ],
+      [ :image , "image", :image_field ],
+      [ :description , "description", :text_area ],
```
Comment (acesuares) [inline app/models/spelling_group.rb]: use text_area_without_ckeditor wherever html layout is not needed!

File: app/models/spelling_group.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ]
+  end
+
+
+  def self.not_accessible_through_html?
+    false
+  end
+
+  def self.order_by_clause
+    "name"
```
Comment (acesuares) [inline app/models/spelling_group.rb]: nil please

File: app/models/word.rb [hunk 3]
```diff
@@ -74,6 +78,8 @@ def inline_forms_attribute_list
       [ :recordings , "recordings", :associated ],
       [ :header_glosario, '', :header ],
       [ :glossaries, '', :check_list ],
+      [ :header_spelspel_grupo, '', :header ],
```
Comment (acesuares) [inline app/models/word.rb]: spel_spel_group ?

File: app/views/spellings/play.html.erb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        <%= hidden_field_tag 'session_id', @spelling_session.id %>
+        <%= text_field_tag 'answer' %>
+      </div>
+      <div class="columns small-12 medium-6 large-3 large-centered text-center">
+        <%= link_to "Stop", spelling_path, class: [:button, :rounded] %>
+      </div>
+    </div>
+  </div>
+</div>
+<div id="myModal" class="reveal-modal tiny" class="reveal-modal" data-reveal>
```
Comment (acesuares) [inline app/views/spellings/play.html.erb]: was this a good idea? is it working for you?

File: config/locales/en.yml [hunk 5]
```diff
@@ -19,6 +19,7 @@ en:
       glossary: glosario
       goal: aréa di edukashon
       source: fuente
+      spelling_group: Grupo di palabra pa Spèl-spèl
```
Comment (acesuares) [inline config/locales/en.yml]: have marta check that word and then add it to the database! I like it very much!

File: config/routes.rb [hunk 6]
```diff
@@ -1,5 +1,15 @@
 Rails.application.routes.draw do
 
+  resources :spelling_groups do
```
Comment (acesuares) [inline config/routes.rb]: dont' need this if it is in MODEL_TABS

File: app/models/spelling_group.rb (hunk 1 above)
Comment (ortegacmanuel) [reply app/models/spelling_group.rb]: Done!

File: config/locales/en.yml (hunk 5 above)
Comment (ortegacmanuel) [reply config/locales/en.yml]: :)

File: app/views/spellings/play.html.erb (hunk 4 above)
Comment (ortegacmanuel) [reply app/views/spellings/play.html.erb]: not compleatly sure if it is a good idea, this was time consuming looking for options and testing, i didn't find  something that convince me about it as a good solution. Finally I decided to put it working this way. Yes, it is working

**Pull request merged**